Pokedex_ios/View/Controller/AdvancedFilterPopup/AdvancedFilterPopupViewController.swift

```diff
@@ -60,15 +62,51 @@ class AdvancedFilterPopupViewController: PopupViewController {
             contentStackView.trailingAnchor.constraint(equalTo: containerView.trailingAnchor, constant: -40)
         ])
     }
+    func resetButtonImages() {
+           viewmodel?.caterpieSelected = false
+           viewmodel?.laprasSelected = false
+           viewmodel?.snorlaxSelected = false
+           weightCaterpieButton.setImage(UIImage(named: "CaterpieUnselected.png"), for: .normal)
```

Comment:
We're also adding all the image names on the ConstantVariables class

Reply:
done